src/api/auth.js

```diff
@@ -30,7 +30,7 @@ export const fetchTokenApi = (username, password) => {
   return sendRequest({
     url,
     method: "POST",
-    credentials: "include",
+    credentials: false,
```

Comment:
Do we really need this change?

Reply:
@GMishx No not necessarily, Ihave fixed it